packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/project/controllers/ViewerProjectsDataFetcher.java

```diff
@@ -78,7 +123,15 @@ private Connection<ProjectDTO> toConnection(Page<ProjectDTO> projectPage) {
         if (!edges.isEmpty()) {
             endCursor = edges.get(edges.size() - 1).getCursor();
         }
-        var pageInfo = new PageInfoWithCount(startCursor, endCursor, projectPage.hasPrevious(), projectPage.hasNext(), projectPage.getTotalElements());
+        boolean hasPreviousPage = false;
```

Comment:
Unfortunately, with the current implementation of Window et KeysetScrollPosition, there is API to get `hasPrevious`.
Furthermore `https://relay.dev/graphql/connections.htm#sec-undefined.PageInfo.Fields` says that If the server can't efficiently determine that value, then it should return false.